packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -83,10 +83,10 @@ class BottomSheet extends StatefulWidget {
     this.constraints,
     required this.onClosing,
     required this.builder,
-  }) : assert(enableDrag != null),
-       assert(onClosing != null),
-       assert(builder != null),
-       assert(elevation == null || elevation >= 0.0);
+  })  : assert(enableDrag != null),
```

Comment:
please turn off auto-formatting for the repo and revert these changes. If using VS Code,

```
    "[dart]": {
        "editor.formatOnSave": false,
        "editor.formatOnType": false
    }
```